skops/hub_utils/tests/test_hf_hub.py

```diff
@@ -303,6 +320,56 @@ def test_init(classifier, config_json):
         )
 
 
+@pytest.fixture(
+    params=[pytest.param("classifier", marks=pytest.mark.usefixtures), get_classifier()]
+)
+def classifiers(request):
+    # Returns a model object or a path to a model with all
+    # model formats combinations from CONFIG dict
+    try:
+        yield request.getfixturevalue(request.param)
+    except Exception:  # get_classifier() is not a fixuture, exception raised
+        yield request.param
+
+
+def test_override_init_modelcard(classifiers, config_json):
+    # create a temp directory and delete it, we just need a unique name.
+    dir_path = tempfile.mkdtemp()
+    shutil.rmtree(dir_path)
+
+    version = metadata.version("scikit-learn")
+    _, model_format = config_json
+    # joblib type falls unders auto format, explicityly set to auto
+    # because we can't repeat key "auto" in CONFIG dict
+    if model_format == "joblib":
+        model_format = "auto"
+
+    init(
+        model=classifiers,
+        requirements=[f'scikit-learn="{version}"'],
+        dst=dir_path,
+        task="tabular-classification",
+        data=iris.data,
+        model_format=model_format,
+    )
+    _validate_folder(path=dir_path)
+
+    # inital card does not have a license set
+    with pytest.raises(
+        AttributeError, match="'CardData' object has no attribute 'license'"
+    ):
+        model_card = RepoCard.load(Path(dir_path) / "README.md")
+        model_card.data.license
+
+    # override existent modelcard created by init with license attribute
```

Comment:
For my better understanding, is this testing some new behavior added by this PR or some general behavior?

Reply:
> we can make this test simpler by checking that the README.md exists after the call to init and that's it?

Yes, that sounds like it is sufficient as a test.